### DISCUSSION TITLE
Account Management

### DISCUSSION THREAD

**Discussion by Prettycharm** [supabase/supabase#8793] [General]
I currently have an app for a client am working on. Is it possible to change account details(username and email) to that of my client. The account information section of the supabase console  currently uses my GitHub data. The two fields seem uneditable. 
How do I transfer content(database, functions, e.t.c) of my account to that of my client? I need to hook up my client's github account with the current supabase account while I remove mine. Thanks.

**Comment by saltcod** [comment]
Hey @Prettycharm 

Not sure if this is enough, but you can change the billing address of a project at: 
https://app.supabase.com/org/[ref]/settings 

You can also transfer a project to a new org but I'm pretty sure you need to be an owner of both orgs. You can submit a support ticket via the Dashboard to do that (the Help button in the top right).
